File: testprojects/csharp/benchmark/Domain/Indirect/AlowedFrom/AccessStaticVariableIndirectIndirect_VarVarVar.cs
namespace Domain.Indirect.AllowedFrom
{
	public class AccessStaticVariableIndirectIndirect_VarVarVar
	{
        private Domain.Indirect.Intermediate.BackgroundService bckg;

		private string test;

		public void Case8()
        {
            bckg = new Domain.Indirect.Intermediate.BackgroundService();
			test = bckg.serviceTwo.serviceOne.sName;
		}
	}
}

File: testprojects/java/benchmark/accuracy-test-2013-04-17/domain/indirect/allowedfrom/AccessStaticVariableIndirectIndirect_VarVarVar.java
package domain.indirect.allowedfrom;

import domain.indirect.intermediate.BackgroundService;

public class AccessStaticVariableIndirectIndirect_VarVarVar {

	private BackgroundService bckg;
	private String test;

	public AccessStaticVariableIndirectIndirect_VarVarVar(){
		bckg = new BackgroundService();
	}
		//  Indirect-indirect access of a static attribute 
		@SuppressWarnings("static-access")
		public String case8() {
			test = bckg.serviceTwo.serviceOne.sName;
			return test;
		}
}